testsuite/MDAnalysisTests/core/test_atomgroup.py

```diff
@@ -967,9 +967,15 @@ def test_n_atoms(self, ag):
 
     def test_n_residues(self, ag):
         assert ag.n_residues == 214
+        
+    def test_zero_atoms_residues(self, ag):
+        ag[[]].residues.atoms
```

Comment:
this test needs some sort of assert to check what is returned, rather than just checking it doesn't cause an `Error`.  something simple like
```python
new_ag = ag[[]].residues.atoms

assert isinstance(new_ag, AtomGroup)
assert len(new_ag) == 0

Reply:
done